src/main/views/sscs-daily-list.njk

```diff
@@ -77,8 +77,12 @@
                           <td class="govuk-table__cell" data-sort-value="{{ sitting['sittingStartFormatted'] | timeToSortValue }}">{{ sitting['sittingStartFormatted'] }}</td>
                           <td class="govuk-table__cell">{{ case['caseNumber'] }}</td>
                           <td class="govuk-table__cell">{{ sitting['caseHearingChannel'] }}</td>
-                          <td class="govuk-table__cell">{{ hearing['applicant'] | addRepresentativeToParty(hearing['applicantRepresentative'], legalAdvisor) }}</td>
-                          <td class="govuk-table__cell">{{ hearing['formattedRespondent'] }}</td>
+                          {% if case['applicant'] | length %}
+                            <td class="govuk-table__cell">{{ case['applicant'] | addRepresentativeToParty(case['applicantRepresentative'], legalAdvisor) }}</td>
+                          {% else %}
+                            <td class="govuk-table__cell">{{ hearing['applicant'] | addRepresentativeToParty(hearing['applicantRepresentative'], legalAdvisor) }}</td>
```

Comment:
If party is at the case level but there is no applicant for that case, hearing['applicant'] will be undefined? This can potentially crash the frontend when displaying the style guide

Reply:
It's never undefined as it gets set to an empty string if there's no applicant. I've added an extra test though to make sure :)